src/content/about/using-cache.md

```diff
@@ -3,143 +3,39 @@ title: Using the Cache
-  - Setting Cloudflare cache rules (i.e. operating on the cf object)
-  - Setting custom cache headers (i.e. Cache-control). This can impact browser as well as Cloudflare Cache behavior.
-- Store responses on a zone scoped namespace separate from Cloudflare's traditional cache using the Cache API from a Workers script. One can control cache behavior of even assets not proxied on Cloudflare.
+- Call to `fetch()` in a Workers script. Requests proxied through Cloudflare are cached even without Workers according to a default or zone’s configured behavior (e.g. static assets like files ending in .jpg are cached by default). Workers can further customize this behavior by:
+  - Setting Cloudflare cache rules (i.e. operating on the `cf` object of a [request](/reference/apis/request/).
+  - Setting custom cache headers (i.e. `Cache-control`). This can impact browsers as well as Cloudflare Cache behavior.
+- Store responses using the Cache API from a Workers script. You can granularly control cache behavior of even assets not proxied on Cloudflare or that would otherwise rely on headers from an origin.
 
 We won't discuss in this article, but other means to control Cloudflare's cache include: Page rules and Cloudflare cache settings. I highly recommend the article [How to Control Cloudflare's cache](https://support.cloudflare.com/hc/en-us/articles/202775670) if you wish to avoid writing Javascript with still some granularity of control.
 
-### [Cache API](/reference/apis/cache)
-
-The Cache API creates a private cache key namespace for each zone using Cloudflare's Global Network. Cache objects are stored in the local datacenter handling the request and are not replicated to other datacenters.
-
-**How is the Cache API different from `fetch()`?** `fetch()` requests a URL and automatically applies caching rules based on your Cloudflare settings. It does not allow you to modify objects before they reach cache, or inspect if an object is in cache before making a request.
-
-The Cache API's `caches.default` Cache object allows you to:
-
-- Fetch a response for a URL if (and only if) it is cached using `caches.default.match(..)`.
-
-- Explicitly store a response in the cache using `caches.default.put(..)` and explicitly delete `caches.default.delete(..)`.
-
-**What are the usage limits on the cache API?**
-
-- 50 total `put()`, `match()`, or `delete()` calls per-request, using the same quota as `fetch()`
-
-- 5 GBs total `put()` per-request
+**What should I use: the Cache API or fetch for caching objects on Cloudflare? **
+For requests where Workers are behaving as middleware (i.e. they are sending a subrequest via `fetch`) it is recommended to use fetch. This is because pre existing settings are in place that optimize caching while preventing unintended dynamic caching. For projects where there is no backend (i.e. the entire project is on Workers as in [Workers Sites](/sites)) the Cache API is the only option for caching.
```

Comment:
I'd recommend "only option to customize caching" instead of "only option for caching". If Workers Sites didn't use the Cache API, it would still benefit to some extent from the caching layer within KV.